agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -116,6 +119,42 @@ async def add_variant_based_on_image(
     )
     await engine.save(db_app_variant)
 
+    # Create testset for demonstration purposes
+    if os.environ["FEATURE_FLAG"] == "demo":
```

Comment:
Let's refactor this to:
1. Have this logic in an another function
2. Make sure that we are adding this to the right application. In the very near future we will have multiple templates. It does not make sense to have this test set for all of these. One easy solution for now is to check for the image id

Reply:
> Have this logic in an another function

Sure, sure.

Reply:
> Make sure that we are adding this to the right application. In the very near future we will have multiple templates. It does not make sense to have this test set for all of these. One easy solution for now is to check for the image id

Yes, yes. Akrem and I discussed this yesterday, thank you!